packages/wrangler/src/index.ts

```diff
@@ -451,6 +452,11 @@ export function createCLIParser(argv: string[]) {
 	});
 
 	// ai
+	wrangler.command("ai", "🤖 Interact with AI models", (aiYargs) => {
+		return ai(aiYargs.command(subHelp));
+	});
+
+	// constellation
```

Comment:
Is the intention to leave in constellation commands?

Reply:
Yes, they will co-exist for now.